Source/Core/Core/PowerPC/ConditionRegister.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/PowerPC/ConditionRegister.h"
+
+const std::array<u64, 16> PowerPC::ConditionRegister::s_crTable = {{
```

Comment:
This files' contents should be placed within a

```cpp
namespace PowerPC
{

}  // namespace PowerPC
```

block instead of writing out `PowerPC::` everywhere it's necessary.